deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -34,6 +34,18 @@
     </regexp>
   </filter>
 {{- end }}
+
+{{ if .Values.otelcol.logLevelFilter }}
+ {{ printf "<filter **%s-otelcol**>" (include "sumologic.fullname" .) }}
```

Comment:
please create or use helper for the name

Reply:
probably `sumologic.metadata.name.otelcol.deployment`

Reply:
```suggestion
 {{ printf "<filter **%s**>" (include "sumologic.metadata.name.otelcol.deployment" .) }}
```

Reply:
@sumo-drosiek Thank you!